packages/ai-chat-ui/src/browser/chat-input-widget.tsx

```diff
@@ -196,6 +207,10 @@ const ChatInput: React.FunctionComponent<ChatInputProperties> = (props: ChatInpu
         };
     }, []);
 
+    React.useEffect(() => () => {
+        props.setEditorRef(undefined);
+    }, []);
```

Comment:
We could place this into the previous `useEffect`, no need to create a new one

Reply:
Right! I'm happy to move it into the existing one. I don't know if it causes overhead, but sometimes I feel adding one `useEffect` per concern is easier to understand. But you know more React than I do, so I'm happy to move it.

Reply:
In general I agree, however here for me both of these `useEffect` have the same topic: life cycle management, i.e. what to do when the component is mounted/unmounted and therefore belong together.